apps/api-server/src/Server.js

```diff
@@ -39,6 +40,24 @@ module.exports  = {
 		});
 	  });
 
+	  this.app.get('/db-health', async (req, res) => {
+		try {
+			await db.sequelize.authenticate();
+			res.status(200).json({
+				status: 'UP',
+				message: 'Database connection has been established successfully.',
+				timestamp: new Date().toISOString(),
+			  });
+		} catch (error) {
+			console.error('Unable to connect to the database:', error);
+			res.status(500).json({
+				status: 'ERROR',
```

Comment:
Looks good! a few things to consider:  getting the errors from a enum or const. Maybe a more descriptive error. Connection_error?

Reply:
Hmm yeah I think it is better to get the error from a enum/const, however, this happens nowhere else in this application, as far as I know of. So doing it like this is more following 'application standards'.

I'm gonna change it to CONNECTION_ERROR, this is better indeed